src/qibo_cloud_backends/braket_client.py

```diff
@@ -0,0 +1,65 @@
+import time
+
+from braket.aws import AwsDevice
+from braket.devices import LocalSimulator
+from qibo import Circuit as QiboCircuit
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes
+
+from qibo_cloud_backends.braket_translation import to_braket
+
+
+class BraketClientBackend(NumpyBackend):
+    def __init__(self, device=None, verbatim_circuit=False, verbosity=False):
+        """Backend for the remote execution of AWS circuits on the AWS backends.
+
+        Args:
+            device (str): The string representing the ARN of the Braket device. Defaults to Braket's statevector LocalSimulator,
+                          LocalSimulator("default"). Other devices are Braket's density matrix simulator,
+                          LocalSimulator("braket_dm"), or any other QPUs.
+            verbatim_circuit (bool): If `True`, to_braket will wrap the Braket circuit in a verbatim box to run it on the QPU
+                                     without any transpilation. Defaults to `False`.
+            verbosity (bool): If `True`, the status of the executed task will be displayed. Defaults to `False`.
+        """
+        super().__init__()
+
+        self.verbatim_circuit = verbatim_circuit
+        self.verbosity = verbosity
+
+        self.device = AwsDevice(device) if device else LocalSimulator()
+        self.name = "aws"
+
+    def execute_circuit(self, circuit_qibo, nshots=1000, **kwargs):
+        """Executes a Qibo circuit on an AWS Braket device. The device defaults to the LocalSimulator().
+
+        Args:
+            circuit (qibo.models.Circuit): circuit to execute on the Braket device.
+            nshots (int): Total number of shots.
+        Returns:
+            Measurement outcomes (qibo.measurement.MeasurementOutcomes): The outcome of the circuit execution.
+        """
+
+        measurements = circuit_qibo.measurements
+        if not measurements:
+            raise_error(RuntimeError, "No measurement found in the provided circuit.")
+        braket_circuit = to_braket(circuit_qibo, self.verbatim_circuit)
+
+        task = self.device.run(braket_circuit, shots=nshots, **kwargs)
+
+        while self.verbosity:
+            status = task.state()
+            print(f"> Status {status}", end=" ", flush=True)
+            if status == "COMPLETED":
+                print("\n")
+                break
+            for _ in range(3):
+                time.sleep(1)
+                print(".", end=" ", flush=True)
+            print("\r" + " " * 30, end="\r")
```

Comment:
Yup. But it might be good to have the monitoring of the status there too.
@BrunoLiegiBastonLiegi and I discussed it here: https://github.com/qiboteam/qibo-cloud-backends/pull/30#discussion_r1764663581

With `verbosity=False` as default, it doesn't print anything. It only prints when `verbosity=True`. 

I think it might be useful to have some way to monitor the task submitted. Usually, the tasks complete their run fairly quickly (few minutes). But if there is a queue, the task will be left in the queue and the user might be left to guess its status if no monitoring is available.

Reply:
Yes, things are getting complicated, because printing it's definitely the easiest.

The correct thing to do would be to delegate the print to the applications' code.
So, you may return the "future" (task), even before resolving, such that it could be probed by the user (that could even be another library).
Then, you would provide a separate function to obtain the `MeasurementOutcomes`, that are essentially the last two lines of this method.

But I acknowledge this is not that trivial. So, maybe better to split this comment to its own issue.

Reply:
> So, you may return the "future" (task), even before resolving, such that it could be probed by the user (that could even be another library).

Ah yes I see your point. That will be helpful indeed, but like you mentioned, it's not so trivial to do.

> But I acknowledge this is not that trivial. So, maybe better to split this comment to its own issue.

Agree too.